packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -37,6 +37,8 @@ type ALUIEvent<T = EventHandlerMap> = ALTimedEvent & ALMetadataEvent & {
     isTrusted: boolean,
     // The underlying identifier assigned to this element
     autoLoggingID: ALID | null,
+    // window.location.href when the event was captured
+    href: string | null;
```

Comment:
I think `href` is more generic than page_uri since in general href can also point to anchors inside of the page, or be data or blob or ...

Isn't this what we eventually always log as `page_uri`? Should we make it more official in our types then?

Reply:
Yes we log it after passing it to `URI.normalize(<val>)`.  That was kept it place but definitely open to changing it here.

>Should we make it more official in our types then?
Do you mean more official in what sense?  Should we put it on all ALLoggable events?  Here was just going to add it to UI events

Reply:
"official" in a sense that we do log it in WWW, but not here. One option is to add `page_uri` to one of the core event types and log it directly in  hyperion.

Reply:
Should I add it to ALLoggableEvent and ensure all events emit this value?

Reply:
Hmm, I think it is better to create a `ALPageEvent` in the `ALTypes` and then add to any payload we want to. 
I'm thinking of when we want to support React Native, and `page_uri` may not have an equivalent there. 
Ideally, this should be a plugin later added for web which adds this field to all events.